terraform/cloudfoundry/security-groups.tf

```diff
@@ -94,7 +94,6 @@ resource "aws_security_group" "web" {
 
     cidr_blocks = [
       "${compact(var.admin_cidrs)}",
-      "${compact(var.tenant_cidrs)}",
```

Comment:
This is because this is about web control panel access and has `web_access_cidrs` on the line below it - previously when this was for `tenant_cidrs` it made sense to give tenants access to both simultaneously, but as this commit aims to make it clearer what the setting is for it didn't make much sense to have `api_access_cidrs` also govern access to web control panels when `web_access_cidrs` already exists for that purpose.